lib/auth/utils.ts

```
@@ -0,0 +1,29 @@
+/** 랜덤 문자열 생성 (Nonce 등) */
+function generateRandomString(length: number = 32): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)